backend/db/refresh.py

```diff
@@ -156,7 +161,7 @@ def refresh_db(
     if check_db_status_var('new_request_while_refreshing'):
         print('INFO: New refresh request detected while refresh was running. Starting a new refresh.')
         delete_db_status_var('new_request_while_refreshing')
-        refresh_db(None,cset_ids,  use_local_db, schema, force_non_contiguity)
+        refresh_db(None, None, use_local_db, schema, force_non_contiguity)
```

Comment:
Bug fix: Was passing cset-ids to secondary refresh (if a user requested a refresh from the web app during that refresh). That isn't supported by the UI.